hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -712,7 +717,17 @@ public void close() {
     }
 
     /**
-     * Clean up any stale/old files/data lying around (either on file storage or index storage)
+     * Clean up any stale/old files/data lying around (either on file storage or index storage) that is past
+     * the typical query timeout. Default is 12 hours.
+     */
+    public void clean() throws HoodieIOException {
```

Comment:
Can we add a metric around how long cleaning takes? It might be really useful going forward to keep an eye on